apps/mobile/src/features/Signers/components/SignersList/SignersListItem.tsx

```diff
@@ -17,13 +17,25 @@ interface SignersListItemProps {
 
 function SignersListItem({ item, index, signersGroup }: SignersListItemProps) {
   const router = useRouter()
+  const local = useLocalSearchParams<{ safeAddress: string; chainId: string }>()
+
   const actions = useSignersActions()
   const isLastItem = signersGroup.some((section) => section.data.length === index + 1)
 
   const onPress = () => {
     router.push(`/signers/${item.value}`)
   }
 
+  const onPressMenuAction = ({ nativeEvent }: NativeActionEvent) => {
+    if (nativeEvent.event === 'import') {
+      let url = '/import-signers'
+      if (local.safeAddress && local.chainId) {
+        url = `/import-signers?safeAddress=${local.safeAddress}&chainId=${local.chainId}`
+      }
+      router.push(url as '/import-signers')
```

Comment:
```suggestion
      router.push('/import-signers', {
        safeAddress: local?.safeAddress,
        chainId: local?.chainId
      })
```

Reply:
I think that you'll run in troubles if you do that. safeAddress and chainId will end up being "undefined" -> like the string undefined. Or maybe not with your approach. When I was using the string I ended up having "undefined" for safeAddress as a string :)